frontend/src/components/blog-details/blog.details.jsx

```diff
@@ -3,37 +3,37 @@ import BlogsService from "pages/api/blogsService";
 import { useState, useEffect } from "react";
 import { useRouter } from "next/router";
 
-export const BlogDetails = ({ blogId }) => {
-	const [blog, setBlog] = useState(null);
+import AuthService from "pages/api/authService";
 
-	const router = useRouter();
+export const BlogDetails = ({ blogId }) => {
+  const [isLoadingBlog, setIsLoadingBlog] = useState(true);
+  const [blog, setBlog] = useState(null);
 
-	useEffect(() => {
-		async function getBlog(blogId) {
-			try {
-				const response = await BlogsService.getBlogById(blogId);
-				setBlog(response.data);
-			} catch (error) {
-				if (error.response.status === 404) await router.replace("/404");
-			}
-		}
+  async function getBlog(token) {

```

Comment:
Sta se ovde desava ako dodje do 404 code-a? Da li next vec ima automatsku 404 stranu ili je potrebno da se ubaci i ta provera ovde u funkciju?

Reply:
Next ima default 404 stranicu (probaj da stavis path u URL koji ne postoji pa ces videti kako izgleda), ali bi bilo bolje kada bi mi implementirali neku custom 404 stranicu. Napravicu issue za to.

Reply:
Super 😀